src/tests/cli_common.py

```diff
@@ -78,7 +82,7 @@ def file_text(path, encoding = CONSOLE_ENCODING):
         return f.read().decode(encoding).replace('\r\r', '\r')
 
 def find_utility(name, exitifnone = True):
-    path = distutils.spawn.find_executable(name)
+    path = shutil.which(name)
```

Comment:
Not tested, but `setuptools._distutils.spawn` provides distutils.spawn and find_executable.
```
# head /usr/lib/python3.11/site-packages/setuptools/_distutils/spawn.py
"""distutils.spawn

Provides the 'spawn()' function, a front-end to various platform-
specific functions for launching another program in a sub-process.
Also provides the 'find_executable()' to search the path for a given
executable name.
"""

```

Reply:
Thanks. Let's use `shutil.which` as it seems to be easier solution. Or do you know any disadvantages?

Reply:
Yes  shutil.which seems the simpler.

LGTM